hack/bump-vcluster-dep.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+const (
+	workflowRelativePath = ".github/workflows/e2e.yaml"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		log.Fatal("please pass target update version (vX.Y.Z, e.g. v0.24.0) as a first argument")
+	}
+	version := os.Args[1]
+	if !strings.HasPrefix(version, "v") {
+		log.Fatal("please pass target update version (vX.Y.Z, e.g. v0.24.0) with 'v' prefix")
+	}
+	workflowBytes, err := os.ReadFile(workflowRelativePath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var output string
+
+	for _, line := range strings.Split(string(workflowBytes), "\n") {
+		if strings.Contains(line, "  VCLUSTER_VERSION:") {
+			replacedLine := "  VCLUSTER_VERSION: v" + version + "\n"
```

Comment:
Wouldn't this result in `vv0.25.0`?

Reply:
yes, thank you for catching that. I initially expected input as `X.Y.Z`, then changed my mind but missed changing this in the script.